### PR TITLE
initial draft of Pilot 3 completion blog post

### PR DESCRIPTION
**Pull request by laxamanaj**
Continue to update and review in #188

### CODE REVIEW

File: vignettes/pilot3-completion-blogpost.qmd [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+2.  Clear documentation on data processing rules and statistical method implementation
+
+3.  Good statistical practice in confirmatory trials, such as avoiding the possibility of “p-hacking” 
+
+For future submissions using open-source languages, it is recommended to give special attention to recommendation theme 1. Recommendation themes 2 and 3 are language-agnostic and should always be followed, regardless of the programming language used.
+
+**Upcoming Pilots**
+
+As a next step, the R Consortium R Submission Working Group initiated submission pilot 4, to explore the use of novel technologies such as Linux containers and web assembly to bundle a Shiny application into a self-contained package, facilitating a smoother process of both transferring and executing the application.
```
Comment (bms63) [inline vignettes/pilot3-completion-blogpost.qmd:92]:
```suggestion
As a next step, the R Consortium R Submission Working Group initiated [submission pilot 4](https://rconsortium.github.io/submissions-pilot4/), to explore the use of novel technologies such as Linux containers and web assembly to bundle a Shiny application into a self-contained package, facilitating a smoother process of both transferring and executing the application.
```

**Review by bms63 — COMMENTED** [review]
I think there is some duplicated sections here with the annoucements

**Comment by robertdevine** [comment]
Cert issue - both ERR

image

We can fix

**Comment by laxamanaj** [comment]
> I think there is some duplicated sections here with the annoucements

removed past announcements to focus on pilot 3.

**Comment by robertdevine** [comment]
@laxamanaj - should be good to go if you git merge --no-edit main and your branch to gather lates. We did some tidying in _**Latest News**_ so you may want to confirm on moving the May 2024 'Latest News' up into the Conferences menu. We moved the announcement of the @bms63 Pharma SUG 2024 presentation in Baltimore. We pulled that from 'Latest News' section and into the menus section of the Pilot3 website with a local pdf link to the @bms63 article to be consistent with PHUSE London and @DeclanHodges under the Conferences menu of the Pilot3 website. Other than that some tidying, a few linter checks, all the CERT fails are addressed. Good to go once all content changes are approved including the latest updates. Thanks again.

**Comment by robertdevine** [comment]
@laxamanaj - successfully tested the RCWG email address provided in the narrative. 
image

**Review by robertdevine — COMMENTED** [review]
@laxamanaj - thank you, to affirm the narrative in review here, as you are aware about the [FDA Clarifying Statement](https://rconsortium.github.io/submissions-pilot3-adam/main/articles/statistical-software-clarifying-statement.html). The RCWG Pilot3 Website does include a link to the FDA statement. The menu item is found under the Project Setup menu on the Pilot3 home page.

**Comment by laxamanaj** [comment]
> @laxamanaj - thank you, to affirm the narrative in review here, as you are aware about the [FDA Clarifying Statement](https://rconsortium.github.io/submissions-pilot3-adam/main/articles/statistical-software-clarifying-statement.html). The RCWG Pilot3 Website does include a link to the FDA statement. The menu item is found under the Project Setup menu on the Pilot3 home page.

Thanks, @robertdevine . Would be good to link this as well!  Thanks.

**Comment by laxamanaj** [comment]
> @laxamanaj - should be good to go if you git merge --no-edit main with your branch to gather latest. We did some tidying in _**Latest News**_ so you may want to confirm on moving the May 2024 'Latest News' item up into the Conferences menu. We moved the announcement of the @bms63 Pharma SUG 2024 presentation in Baltimore. We pulled that from 'Latest News' section and into the menus section of the Pilot3 website with a local pdf link to the @bms63 article to be consistent with PHUSE London and @DeclanHodges under the Conferences menu of the Pilot3 website. Other than that some tidying, a few linter checks, all the CERT fails are addressed. Good to go once all content changes are approved including the latest updates. Thanks again.

Hi @robertdevine . Sorry I haven't done the merge from main yet. I will do it now with main, then merge this back to main for final merge. Then will deliver draft blog post to Joe et al for review.

**Comment by robertdevine** [comment]
Thank you, @laxamanaj I did share on LinkedIn the **_R consortium R submission pilot 3 to FDA (R based ADaM code submission)_** video that Ning posted earlier today. Your presentation captures a wonderful overview. Great team effort! What a Team from the Industry Sponsors and of course Regulatory (FDA) and kudos to RCWG for hosting and organizing the ongoing efforts! Truly a positive impact and many are starting to look at the RCWG pilots to inform their work. Thank you again.

**Pull request merged**